### PR TITLE
Fix api tests

### PR DESCRIPTION
**Pull request by snayrouz**
#

Description of Changes Made: 
API for stores works. We can use that for that main page json to show all stores and show one store by ID.
Anticipated Code Effects: 
(Would this merge affect any other piece of our code?)
Tests pass
Reviewer:

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ group :development, :test do
   gem 'shoulda-matchers'
   gem 'launchy'
   gem 'faker'
-  gem 'factory_girl'
```
Comment (jmejia) [inline Gemfile:41]: We should just delete instead of commenting out.

Comment (snayrouz) [reply Gemfile:41]: ya i think so too. I was running into factory girl issues so I switched the gem and configurations to factory_bot_rails, but no one else was having the same issues as me so we're going to see what works best for everyone before merging this

**Review by jmejia — COMMENTED** [review]
I could be wrong but it doesn't look like the API is being authenticated with an API key. It looks like it using a session which isn't available outside of the browser.

**Comment by snayrouz** [comment]
Closing this because it looks like you all did this already?

**Pull request closed without merging**